package.json

```diff
@@ -20,10 +22,17 @@
   "license": "AGPL-3.0",
   "keywords": [
     "node",
-    "glitch",
-    "express"
+    "express",
+    "chat"
   ],
   "devDependencies": {
-    "eslint-config-node": "^4.0.0"
-  }
+    "eslint-config-node": "^4.0.0",
+    "common-good": "^2.0.3",
+    "stylelint-config-recommended": "^3.0.0"
+  },
+  "author": "Christian Bundy",
```

Comment:
I could have my name here... someday :P

Reply:
I'd 100% merge a patch turning that into `contributors` instead of `author`.